Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM python:2.7-slim
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+		dos2unix \
+		gcc \
+		libc-dev \
+	&& rm -rf /var/lib/apt/lists/*
+
+ENV TINI_VERSION v0.13.2
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+RUN chmod +x /tini
+
+ENV RQ_VERSION 0.7.1
+RUN pip install rq==$RQ_VERSION
+
+ADD autoexec.sh /
+RUN dos2unix /autoexec.sh
```

Comment:
Agreed it would be better, but I'm building on Docker for Windows, which makes juggling CRLF a bit of a minefield. Every now and then you have something reset the git options and/or use the "wrong" text editor and suddenly you have some CRLF in there causing the script not to run. 

So I felt it was much safer and user friendly to simply fix it inside the container, making the build process more robust.